ddtrace/internal/ci_visibility/_api_client.py

```diff
@@ -80,6 +87,13 @@ class EarlyFlakeDetectionSettings:
     faulty_session_threshold: float = 30.0
 
 
+@dataclasses.dataclass(frozen=True)
+class UniqueTestsData:
+    test_modules: t.Dict[str, TestModuleId]
+    test_suites: t.Dict[str, TestSuiteId]
+    test_cases: t.Dict[str, InternalTestId]
```

Comment:
"test case" here means individual tests? I ask because `unittest` usually uses a `BlaBlaTestCase` class for a test suite (if I understand correctly), so this might be confusing.

Reply:
(resolving because I'm deleting)